src/VirtualDesktop/Interop/Build22621/.interfaces/IVirtualDesktopNotification.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using WindowsDesktop.Interop.Build10240;
+
+namespace WindowsDesktop.Interop.Build22000
```

Comment:
Namespace should match the build:
namespace WindowsDesktop.Interop.Build22621

Reply:
Sorry, I missed that. I'm fixing those and I am testing everything before my next commit.